src/org/zaproxy/zap/extension/exportreport/PanelAlertDetails.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport;
+
+import java.awt.BorderLayout;
+import java.awt.FlowLayout;
+import java.util.ArrayList;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.SpringLayout;
+
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.extension.exportreport.Utility.DualListBox;
+import org.zaproxy.zap.extension.exportreport.Utility.SharedFunctions;
+import org.zaproxy.zap.extension.exportreport.Utility.SpringUtilities;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ */
+
+public class PanelAlertDetails extends JPanel
+{
+	private DualListBox				dual			= null;
+	private ExtensionExportReport	extension		= null;
+	ArrayList<String>				list			= null;
+	String[]						arr;
+	ArrayList<String>				listExcluded	= null;
+	String[]						arrExcluded;
+
+	public PanelAlertDetails(ExtensionExportReport extension, ArrayList<String> list, ArrayList<String> listExcluded)
+	{
+		this.extension = extension;
+		this.list = list;
+		this.arr = new String[this.list.toArray().length];
+		this.listExcluded = listExcluded;
+		this.arrExcluded = new String[this.listExcluded.toArray().length];
+
+		initialize();
+	}
+
+	private void initialize()
+	{
+		String strLabel = Constant.messages.getString("exportreport.menu.details.label");
+		JLabel lblLabel = null;
```

Comment:
It is being used `lblLabel = SharedFunctions.createLabel(top, lblLabel, strLabel, SharedFunctions.getTitleFont());`